test/testcafe/spec/DeviceProbingChromeDTC_spec.js

```diff
@@ -8,13 +8,13 @@ import identifyWithDeviceProbingLoopback from '../../../playground/mocks/data/id
 
 const mockChromeProbingView = RequestMock()
   .onRequestTo(/\/idp\/idx\/introspect/)
-  .respond((req, res) => {
+  .respond((_, res) => {
```

Comment:
Was this causing a linting error for you?

Reply:
I think I thought it was but I just re-added and it passes. Do we have a preference to leave `req` in there for readability even if they're always unused variables?

Reply:
I think either way is fine, and we do have a linting exception to allow the `_` as a placeholder when you need to skip a parameter like this